src/domain/tokens/getTotalTokenBalance.ts

```diff
@@ -0,0 +1,22 @@
+import { TokensData } from "domain/synthetics/tokens/types";
+import { convertToUsd } from "domain/synthetics/tokens/utils";
+
+export function getTotalTokenBalance(tokensData: TokensData | undefined, tokenSymbol: string) {
+  const defaultResult = {
+    balance: 0n,
+    balanceUsd: 0n,
+  };
+
+  if (!tokensData) {
+    return defaultResult;
+  }
+
+  const tokens = Object.values(tokensData).filter((token) => token.symbol === tokenSymbol);
```

Comment:
hmm, a bit strange api, it accepts tokensData and only signle tokenSymbol?
Maybe let's add symbols array at least